### ISSUE TITLE
PPO on Ant-v3 giving unexpected results

### ISSUE THREAD

**Issue by arnavc1712** [tensorforce/tensorforce#874]
Hi,
I am trying to recreate OpenAI's PPO training using Tensorforce. I have tried to replicate the hyperparameters used in [https://spinningup.openai.com/en/latest/spinningup/bench.html](url) but:
1. I am not being able to reach the same reward as mentioned in the URL (~2000)
2. What the policy learns during training vs evaluation looks very different (when visualizing episodes during training and testing)
3. During validation the reward seems to be ~1000, but all the Ant seems to be doing is staying in one place.

Agent:
```
network_spec = [
    dict(type='dense', size=64, activation='tanh'),
    dict(type='dense', size=32, activation='tanh'),
]
bs_network_spec = [
    dict(type='dense', size=64, activation='tanh'),
    dict(type='dense', size=32, activation='tanh'),
]

agent = Agent.create(
    agent='ppo', 
    environment=environment, 
    batch_size=4, 
    learning_rate=3e-4, 
    exploration=0.05,
    likelihood_ratio_clipping=0.2,
    network=network_spec,
    subsampling_fraction=1.0,
    discount=0.99,
    multi_step=80,
    baseline=bs_network_spec,
    baseline_optimizer=dict(
        optimizer='adam', learning_rate=1e-3,
        multi_step=80, subsampling_fraction=1.0,
    )
)
```

I am not sure what is going wrong here. Is the reward calculation wrong? Is the agent being used during train/test different? 
Any help would be appreciated!

Here is the code to my colab link: [https://colab.research.google.com/drive/1Hpi_DFyxMfTDoQ1iUtwz1epHAziQo1HI?usp=sharing](url)

Video of trained Agent interacting with Ant-v3: [https://drive.google.com/file/d/1p514dg5oEUOUcGnM5A8kXa40Qo1Wbv7g/view?usp=sharing](url)

Episodic Rewards per training step:
![image](https://user-images.githubusercontent.com/19833834/174219645-8c5d5ef8-5e32-42ef-8841-3e7bb8e5fbd8.png)
